source/browseMode.py

```diff
@@ -2367,6 +2367,14 @@ def _iterTextStyle(
 			direction: documentBase._Movement = documentBase._Movement.NEXT,
 			pos: textInfos.TextInfo | None = None
 	) -> Generator[TextInfoQuickNavItem, None, None]:
+		from NVDAObjects.window.winword import BrowseModeWordDocumentTextInfo
+		if isinstance(pos, BrowseModeWordDocumentTextInfo):
+			raise NotImplementedError(
+				"We don't support non-UIA word textInfos due to multiple issues with textInfo implementation "

```

Comment:
Are there NVDA GitHub issues describing these text info problems?
If yes, could you reference them here?

Reply:
I think referencing this PR makes more sense here given all the issues linked in this PR description

```suggestion
				"We don't support non-UIA word textInfos due to multiple issues with textInfo implementation - #16569"
```